File: Examples/CSharp/CustomizingCreatedProject/CreateEmptyProjectSaveXML.cs
using Aspose.Tasks.Saving;

namespace Aspose.Tasks.Examples.CSharp.CustomizingCreatedProject
{
    public class CreateEmptyProjectSaveXML
    {
        public static void Run()
        {
            // ExStart:CreateEmptyProjectSaveXML
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_CustomizingCreatedProject();
            Project project = new Project();
            project.Save(dataDir+ "saved_out.xml", SaveFileFormat.XML);
            // ExEnd:CreateEmptyProjectSaveXML
        }
    }
}

File: Examples/VisualBasic/CustomizingCreatedProject/CreateEmptyProjectSaveXML.vb
Imports Aspose.Tasks.Saving

Namespace Aspose.Tasks.Examples.VisualBasic.CustomizingCreatedProject
    Public Class CreateEmptyProjectSaveXML
        Public Shared Sub Run()
            'ExStart: CreateEmptyProjectSaveXML
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_CustomizingCreatedProject()
            Dim project As New Project()
            project.Save(dataDir & "saved.xml", SaveFileFormat.XML)
            'ExEnd: CreateEmptyProjectSaveXML
        End Sub
    End Class
End Namespace